packages/studio/src/components/PreviewWithUseComponents.tsx

```diff
@@ -4,33 +4,45 @@ import PreviewPanel from "./PreviewPanel";
 import Highlighter from "./Highlighter";
 import IFramePortal from "./IFramePortal";
 import { CSSProperties } from "react";
+import { useState } from "react";
+import { Tooltip } from "react-tooltip";
 
 const inlineStyles: CSSProperties = {
-  transform: "translateY(-50px)",
-};
-
-const innerIframeStyles: CSSProperties = {
-  transform: "translateY(50px)",
+  overflow: "scroll",
 };
 
 export default function PreviewWithUseComponents() {
   const componentTree = useStudioStore((store) =>
     store.actions.getComponentTree()
   );
+  const [tooltipProps, setTooltipProps] = useState({
+    open: false,
+    position: { x: 0, y: 0 },
+    error: "",
+    anchorId: "",
+  });
   void useImportedComponents(componentTree);
 
   return (
-    <IFramePortal
-      className="h-full w-full"
-      title="PreviewPanel"
-      inlineStyles={inlineStyles}
-    >
-      <div style={innerIframeStyles}>
-        <PreviewPanel />
-        <div className="absolute top-[-50px]">
+    <>
+      <Tooltip
+        offset={-30}
+        content={tooltipProps.error}
+        anchorSelect={tooltipProps.anchorId}
+        className="text-sm z-20"
+        isOpen={tooltipProps.open}
+        position={tooltipProps.position}
```

Comment:
nit: can we spread the tooltipProps here instead of individually listing out each one?